uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html

```diff
@@ -74,14 +75,14 @@
             </div>
             <!-- Subject -->
             <div class="row">
-                <div class="col-xs-12 col-sm-6" ng-show="!showFieldSubject">
+                <div class="col-xs-12 col-sm-6" ng-show="!task.subject && !showFieldSubject">
                     <span class="form-control-static">
                         <a href class="btn btn-indent" ng-click="showFieldSubject = true;">
```

Comment:
i don't know but i think it is better for href to have a value .. something like href="javascript:void(0);"